### PR TITLE
Implement color theming support

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #39 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update dependencies
* Implement color theming support with tier 3 tokens

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jordanjones243** [comment]
Couple things to note here:
- The "hyperlinks" are styled using the `--ds-auro-hyperlink-visited...`. Why is this the rule/state token that is being applied without interaction with the component?
- The border around the breadcrumbs when clicked appear to have a color, but before they were transparent. What is the correct UI?

File: src/auro-nav.js [hunk 1]
```diff
@@ -292,14 +311,14 @@ export class AuroNav extends LitElement {
         <slot @slotchange="${this.handleSlotItems}"></slot>
       </div>
       ${this.anchorlinks && this.anchorlinks.length > this.mobileViewCollapsedNumLinks ? html`
-        <auro-button slim tertiary class="showHideBtn" @click=${this.toggleAnchorLinks}>
+        <${this.buttonTag} slim tertiary class="showHideBtn" @click=${this.toggleAnchorLinks} id="anchorlinkButton">
```
Comment (jason-capsule42) [inline src/auro-nav.js:314]: It looks like the ID is only added to make a test pass. Can't you just make the test selector look for the class `showhideBtn` that is already on the element?

File: src/style-anchorlink.scss [hunk 2]
```diff
@@ -3,22 +3,24 @@
...
-
 :host {
   display: block;
-  border: 1px solid transparent;
-  border-left: 1px solid var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  border-width: 1px;
+  border-style: solid;
+
+  border-left-width: 1px;
+  border-left-style: solid;
```
Comment (jason-capsule42) [inline src/style-anchorlink.scss:20]: why re-declare these when they are technically already set with the same values on line 16-17?

File: src/style-anchorlink.scss [hunk 3]
```diff
@@ -28,26 +30,17 @@
...
     }
   }
-}
 
-:host([active]),
-:host(:hover) {
-  a {
-    color: var(--ds-color-text-primary-default, $ds-color-text-primary-default) !important;
+  .hyperlink {
+    white-space: unset;
```
Comment (jason-capsule42) [inline src/style-anchorlink.scss:38]: What was the reason we needed to add this?

File: src/style-breadcrumb.scss [hunk 4]
```diff
@@ -48,18 +47,17 @@
...
 
   a {
-    color: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
     text-decoration: none; // stylelint-disable-line no-eol-whitespace
   }
 
   @include auro_breakpoint($max: $ds-grid-breakpoint-sm) {
     // Displays left chevron icon
-    ::slotted(auro-icon[name='chevron-left']) {
+    ::slotted([auro-icon][name='chevron-left']) {
```
Comment (jason-capsule42) [inline src/style-breadcrumb.scss:55]: why can't we just do this off the name attribute selector without including `[auro-icon]`? Selectors with multiple identifiers are slower.

File: src/style-breadcrumb.scss [hunk 5]
```diff
@@ -48,18 +47,17 @@
...
       display: inline-block;
     }
 
     // Hides the home icon for all breadcrumbs
-    ::slotted(auro-icon[name='home-stroke']) {
+    ::slotted([auro-icon][name='home-stroke']) {
```
Comment (jason-capsule42) [inline src/style-breadcrumb.scss:60]: Same selector comment as line 55.

File: src/style-breadcrumb.scss [hunk 6]
```diff
@@ -71,15 +69,13 @@
 
 // Hides the home icon for all breadcrumbs besides the first
 :host(:not(:first-of-type)) {
-  ::slotted(auro-icon[name='home-stroke']) {
+  ::slotted([auro-icon][name='home-stroke']) {
```
Comment (jason-capsule42) [inline src/style-breadcrumb.scss:72]: Same selector comment as line 55.

File: src/style-anchorlink.scss (hunk 3 above)
Comment (jordanjones243) [reply src/style-anchorlink.scss:38]: This was done because the auro-hyperlink were causing for text not to wrap in anchornav

Comment (jason-capsule42) [reply src/style-anchorlink.scss:36]: Just add a comment saying as much in the CSS. We aren't going to remember that in 6 months.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**